guides/v2.3/graphql/reference/quote.md

```diff
@@ -436,6 +436,15 @@ Attribute |  Data Type | Description
 `current_balance` | Money | Current balance remaining on the gift card
 `expiration_date` | String | Gift card expiration date
 
+#### AppliedStoreCredit object {#AppliedStoreCredit}
+
+The `AppliedStoreCredit` object can contain the following attributes:
```

Comment:
```suggestion
The `AppliedStoreCredit` object can contain the following attributes.
```

Reply:
The colon is consistent with all the other lead-in sentences in this topic.